Lesson_7/Lesson_7/ICoder.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lesson_7
+{
+    interface ICoder
+    {
+        public string Encode(string filePathToCode, string encodedFilePath);
+        public string Decode(string filePathToCode, string decodedFilePath);
+    }
```

Comment:
```suggestion
        string Encode(string filePathToCode, string encodedFilePath);
        string Decode(string filePathToCode, string decodedFilePath);
    }
```
В интерфейсе всё публичное. Модификатор доступа обычно не ставится.